pep/app/constants/text.ts

```diff
@@ -0,0 +1,76 @@
+export enum FontSizes {
+    XS = 'xs',
+    SMALL = 'small',
+    DEFAULT = 'default',
+    LARGE = 'large',
+    XLARGE = 'xlarge'
+}
```

Comment:
i think we may have strayed from this, but the project was originally following the convention of naming enums in the singular form, e.g. `FontSize`, looks like most of the enums follow that. Imo it just reads a bit better when using them in typing args, etc, as like `(newSize: FontSizes)` reads as if you are passing it multiple font sizes.